.appveyor.yml

```diff
@@ -9,7 +9,7 @@ environment:
   PHPCI_COMPOSER: C:\tools\phpci\composer
 
 cache:
-  - '%PHPCI_CACHE%'
+  - '%PHPCI_CACHE% -> appveyor.yml'
```

Comment:
What does this do?

Reply:
Makes the cache depending on the appveyor.yml file. If the file is changed the cache gets invalidated.

Reply:
Nice. Should we also update the PHP version to 7.3.1?

Reply:
👍 yes, but probably not in this PR ☺️

Reply:
Done in 5f1c5bea31f9b82886af647a889012f122b1fb3e